rules/ml/ml_rare_destination_country.toml

```diff
@@ -0,0 +1,38 @@
+[metadata]
+creation_date = "2021/04/05"
+maturity = "production"
+updated_date = "2021/04/05"
+
+[rule]
+anomaly_threshold = 75
+author = ["Elastic"]
+description = """
+A machine learning job detected an unusual destination country name in the network logs.
+This can be due to initial access, persistence, command-and-control, or exfiltration activity.
+For example, when a user clicks on a link in a phishing email or opens a malicious document, 
+a request may be sent to download and run a payload from a server in a country which does not 
+normally appear in network traffic or business work-flows. Malware instances and persistence 
+mechanisms may communicate with command-and-control (C2) infrastructure in their country of origin, 
+which may be an unusual destination country for the source network. 
+"""
+false_positives = [
+    """
+    Business workflows that occur very occasionally, and involve a business relationship with an 
+    organization in a counry that does not routinely appear in network events, can trigger this alert.
+    A new business workflow with an organization in a country with which no workflows previosuly 
+    existed may triggger this alert - although the model will learn that the new destination coutnry 
+    is no longer anomalous as the activity becomes ongoing. Business travelers who roam to many 
+    countries for brief periods may trigger this alert. 
+    """,
+]
+from = "now-30m"
+interval = "15m"
+license = "Elastic License"
+machine_learning_job_id = "rare-destination-country"
+name = "Rare Destination Country"
```

Comment:
I think some of these rule names can be more descriptive for analysts when they see a new alert. E.g. Instead of `Rare Destination Country`, we could say `Network Traffic to Rare Destination Country`.

Reply:
Changed some names.